crates/sui-bridge-indexer/src/eth_bridge_indexer.rs

```diff
@@ -411,68 +415,199 @@ impl DataMapper<RawEthData, ProcessedTxnData> for EthDataMapper {
+                        log.block_number()
+                    );
+
+                    processed_txn_data.push(ProcessedTxnData::GovernanceAction(GovernanceAction {
+                        tx_digest: txn_hash.clone(),
+                        sender: txn_sender.clone(),
+                        timestamp_ms,
+                        action: GovernanceActionType::UpgradeEVMContract,
+                        data: serde_json::to_value(bridge_event).unwrap(),
+                    }));
+                }
+                EthBridgeLimiterEvents::InitializedFilter(_)
+                | EthBridgeLimiterEvents::HourlyTransferAmountUpdatedFilter(_)
+                | EthBridgeLimiterEvents::OwnershipTransferredFilter(_) => {}
+            },
+            EthBridgeEvent::EthBridgeConfigEvents(bridge_event) => match bridge_event {
+                EthBridgeConfigEvents::TokenPriceUpdatedFilter(_) => {
+                    info!(
+                        "Observed Eth TokenPrices Update at block: {}",
+                        log.block_number()
+                    );
+
+                    processed_txn_data.push(ProcessedTxnData::GovernanceAction(GovernanceAction {
+                        tx_digest: txn_hash.clone(),
+                        sender: txn_sender.clone(),
+                        timestamp_ms,
+                        action: GovernanceActionType::UpdateTokenPrices,
+                        data: serde_json::to_value(bridge_event).unwrap(),
+                    }));
+                }
+                EthBridgeConfigEvents::TokenAddedFilter(_) => {
+                    info!("Observed Eth AddSuiTokens at block: {}", log.block_number());
+
+                    processed_txn_data.push(ProcessedTxnData::GovernanceAction(GovernanceAction {
+                        tx_digest: txn_hash.clone(),
+                        sender: txn_sender.clone(),
+                        timestamp_ms,
+                        action: GovernanceActionType::AddEVMTokens,
+                        data: serde_json::to_value(bridge_event).unwrap(),
+                    }));
+                }
+                EthBridgeConfigEvents::UpgradedFilter(_) => {
+                    info!(
+                        "Observed Eth BridgeConfig Upgrade at block: {}",
+                        log.block_number()
+                    );
+
+                    processed_txn_data.push(ProcessedTxnData::GovernanceAction(GovernanceAction {
+                        tx_digest: txn_hash.clone(),
+                        sender: txn_sender.clone(),
+                        timestamp_ms,
+                        action: GovernanceActionType::UpgradeEVMContract,
+                        data: serde_json::to_value(bridge_event).unwrap(),
+                    }));
+                }
+                EthBridgeConfigEvents::InitializedFilter(_) => {}
+            },
+            EthBridgeEvent::EthCommitteeUpgradeableContractEvents(_) => {}
```

Comment:
we don't care about this?

Reply:
no we're capturing a different upgrade event

Reply:
what happens when this event gets emitted? should we scream?

Reply:
I have added warn log for all the unexpected events.